Card/src/main/java/com/paypal/android/card/CardClient.kt

```diff
@@ -1,6 +1,30 @@
 package com.paypal.android.card
 
-import com.paypal.android.core.PaymentsClient
+import com.paypal.android.card.network.CardAPIClient
+import com.paypal.android.core.APIClient
+import com.paypal.android.core.PaymentsConfiguration
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
```

Comment:
I'm getting a missing dependency error on these imports, is my Android Studio wrong or did we lose a dependency somewhere?

Reply:
I think this was intermittent, hopefully should be good now.